libraries/Zigbee/src/ZigbeeCore.h

```diff
@@ -118,6 +119,10 @@ class ZigbeeCore {
     return _scan_duration;
   }
 
+  void setRxOnWhenIdle(bool rx_on_when_idle) {
```

Comment:
If you go with global `_rx_on_when_idle`, you would also need to implement this in the source file